src/renderers/shared/fiber/ReactDebugCurrentFiber.js

```diff
@@ -34,7 +34,11 @@ function getCurrentFiberOwnerName() : string | null {
     switch (fiber.tag) {
       case IndeterminateComponent:
       case ClassComponent:
+        return getComponentName(fiber);
       case FunctionalComponent:
+        if (fiber._debugOwner != null) {
+          return getComponentName(fiber._debugOwner);
+        }
```

Comment:
Right, but this doesn’t mean functional components are special compared to class components, does it? I think this might mean we need separate methods, e.g. `getCurrentFiberOwnerName` and `getCurrentFiberName`. Or something like this. Maybe we’re wrong in using `getCurrentFiberOwnerName` for both of these errors, and one of them needs something different.

I haven’t really thought about this much, just giving you ideas to think about. It’s more likely that these errors need different Fibers in messages, than that functional components are special.